### PR TITLE
Update description of video.js in the package.json file, and add 'hls' keyword

### PR DESCRIPTION
**Pull request by OwenEdwards**
## Description
Just a suggestion; update the description of video.js to better reflect what it is now, rather than what it was. It seems like this change should have happened when v7 was first released, but maybe it makes sense to do it now?

### CODE REVIEW

**Comment by gkatsev** [comment]
We also support most of DASH. Maybe we should add HLS and DASH to tags but not specifically call out HLS in the description?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "video.js",
-  "description": "An HTML5 and Flash video player with a common API and skin for both.",
+  "description": "An HTML5 and HLS video player (with optional Flash support) with a common API and skin.",
```
Comment (heff) [inline package.json:3]: How about this?

```suggestion
  "description": "An HTML5 video player that supports HLS, DASH, and Flash, with a common API and skin.",
```

**Comment by OwenEdwards** [comment]
@gkatsev and @heff - since HLS (and DASH) support is now in the core of video.js, and Flash support has been pushed out into a plug-in, shouldn't the description make that distinction clear?

**Comment by gkatsev** [comment]
I'm OK just striking Flash from the record given that in a couple of months it won't be a thing.

**Comment by mister-ben** [comment]
And even mentioning Flash just encourages "how do I play RTMP?"

**Pull request merged**